.eslintrc

```diff
@@ -30,6 +30,7 @@
     "unicorn/filename-case": "off",
     "unicorn/import-style": "off",
     "unicorn/no-abusive-eslint-disable": "off",
+    "unicorn/no-array-callback-reference": "off",
```

Comment:
We can now pass functions directly to array methods.
Neccesary before:
```anArray.map(thing => aFunc(thing))```
now can do ^ or:
```anArray.map(aFunc)```

Reply:
+1